src/cartridges/int_adyen_SFRA/cartridge/client/default/js/paypalExpress.js

```diff
@@ -0,0 +1,52 @@
+// const helpers = require('./adyen_checkout/helpers');
+const { checkIfExpressMethodsAreReady } = require('./commons/index');
+const { updateLoadedExpressMethods } = require('./commons');
+
+const PAYPAL = 'paypal';
+
+let checkout;
+// let shippingMethodsData;
+
+async function initializeCheckout() {
+  const session = await fetch(window.sessionsUrl);
+  const sessionData = await session.json();
+
+  // const shippingMethods = await fetch(window.shippingMethodsUrl);
+  // shippingMethodsData = await shippingMethods.json();
+
+  checkout = await AdyenCheckout({
+    environment: window.environment,
+    clientKey: window.clientKey,
+    locale: window.locale,
+    session: sessionData,
+  });
+}
+
+initializeCheckout().then(() => {
+  const payPalPaymentMethod =
+    checkout.paymentMethodsResponse.paymentMethods.find(
+      (pm) => pm.type === PAYPAL,
+    );
+
+  if (!payPalPaymentMethod) {
```

Comment:
We could change this to a positive condition and include line 37 to line 49 in IF block. This way line 32 and 33 would not be repeated as line 50 and 51